models/topics.models.js

```diff
@@ -0,0 +1,9 @@
+const db = require("../db/connection");
+
+function fetchTopics() {
+  return db.query("SELECT * FROM topics;").then((topics) => {
+    return topics;
```

Comment:
why not destructure rows here so that what you send back is the actual array of topics instead of the response from the query